### DISCUSSION TITLE
tenant discovery

### DISCUSSION THREAD

**Discussion by v-kessler** [ory/kratos#3056] [Q&A]
Situation:
1. saas.com Application
2. Each Company has company1.saas.com company2.saas.com
3. per customer SaaS creates an isolated postgres db /customer1 and customer2 and isolated kratos
4. user max@example.com is registered  in both  /customer1 and customer2. 
5. Login screen on saas.com additionally to user/pwd or Oauth requires from user information which customer to root to appropriate IdP 

Question:
Is there a possibility to avoid to ask user which customer but instead allow to login and then offer him two customers to choose from. Some sort of home realm discovery or user federation cross tenants?

**Comment by aeneasr** [comment]
Hi there, Ory Kratos does not support multi tenancy when self-hosting out of the box. See: https://www.ory.sh/docs/kratos/guides/multi-tenancy-multitenant

We do however support multi tenancy on the Ory Network!
